HW3/classifier.py

```diff
@@ -18,13 +19,18 @@ def train_classifier(features, labels, args):
 
 
 def make_predictions(clf, features):
-    prediction_probs = clf.predict_proba(features)
-    return np.argmax(prediction_probs, axis=-1) + 1, \
-        np.max(prediction_probs, axis=-1)
+    return clf.predict(features)
```

Comment:
Scores could be used for finding the most cat-like dog, the most dog-like cat, and the most dog-like dog and the most cat-like cat (for visualizing false predictions).

Reply:
True, I haven't gotten to that part of the assignment yet. In that case  it needs 
clf = SVC(C=args.svm_lambda, probability=True)

Reply:
Yes, you're right.